src/documentation.ts

```diff
@@ -68,8 +68,19 @@ export class Documentation extends OpenApiBuilder {
       : undefined;
   }
 
-  protected ensureUniqOperationId(path: string, method: Method) {
-    const operationId = makeCleanId(path, method);
+  protected ensureUniqOperationId(
+    path: string,
+    method: Method,
+    endpointOperationId?: string,
+  ) {
+    if (
+      endpointOperationId &&
+      endpointOperationId in this.lastOperationIdSuffixes
+    ) {
+      throw new Error(`Duplicated operationId: "${endpointOperationId}"`);
```

Comment:
this should be `DocumentationError`

Reply:
(from `/src/errors`)

Reply:
DocumentationError's constructor also provides information on the endpoint caused the issue, could be useful